ui/app/rif/components/chainAddresses.js

```diff
@@ -183,6 +183,14 @@ class ChainAddresses extends Component {
   async addAddress (address = null, chainAddress = null, toastMessage = 'Adding chain address', action = 'add') {
     const insertedAddress = address || this.state.insertedAddress;
     const selectedChainAddress = chainAddress || this.state.selectedChainAddress;
+    // Before sending the new chainaddress to be added/updated, we need to check if the user is not trying to set something that already has
+    if (action === 'add' && this.state.chainAddresses.find(chAddress => chAddress.chain === selectedChainAddress)) {
+      this.props.showToast('Can\'t add an address that\'s already setted. Try updating it from the list', false);
```

Comment:
"Address is already present. Try updating it from the list or add a new one" seems better to me